src/include/duckdb/core_functions/scalar/date_functions.hpp

```diff
@@ -490,8 +490,8 @@ struct ToSecondsFun {
 struct ToTimestampFun {
 	static constexpr const char *Name = "to_timestamp";
 	static constexpr const char *Parameters = "sec";
-	static constexpr const char *Description = "Converts sec since epoch to a timestamp";
-	static constexpr const char *Example = "to_timestamp(701222400)";
```

Comment:
See line 8 - `// This file is generated by scripts/generate_functions.py` - these changes need to be made in the associated JSON files as well

See https://github.com/duckdb/duckdb/blob/a83afa7cc3d351afd9793a3b01df0a62f05e988b/src/core_functions/scalar/date/functions.json#L360